tr/about/website/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: "Ruby Websitesi Hakkında"
+lang: tr
+---
+
+Bu websitesi, [Jekyll][jekyll] kullanılarak Ruby ile oluşturulmuştur,<br>
+kaynağı [GitHub][github-repo]'da barındırılmaktadır.
+
+Görsel tasarım [Jason Zimdars][jzimdars] tarafından yapılmıştır.<br>
+Ruby Visual Identity Team'in daha önceki bir tasarımı tabanlıdır.
```

Comment:
Aslında burada şöyle yazsak bence daha güzel ve anlaşılır olur. `Bu görsel tasarım Ruby Visual Identity Team'in daha önceki tasarımını örnek Jason Zimdars tarafından yapılmıştır.`

Reply:
Bu cümlede anlatım bozukluğu yok mu? Ya da ben mi anlayamadım.

Reply:
evet eksik yazmışım. `Bu görsel tasarım Ruby Visual Identity Team'in daha önceki tasarımını örnek alarak Jason Zimdars tarafından yapılmıştır.` gibi olabilir şeklinde düşünüyordum.

Reply:
İlk cümlenin sonundaki `<br>` dikkatinizi çekti mi? Sanki önce Jason Zimdars'a, daha sonra Ruby Visual Identity Team'e vurgu yapılmak istenmiş gibi. Bu durumda bu sıralamaya uyan bir ifade daha iyi olmaz mı? Ayrıca tek cümle ile sanki biraz karmaşık oluyor ifade.